BackEndFlask/Functions/addStudentsAndAssignTeams.py

```diff
@@ -0,0 +1,304 @@
+from typing import List, Tuple, TextIO
+
+from Functions.helper import helper_verify_email_syntax, helper_cleanup, helper_create_user
+from Functions.customExceptions import *
+from Functions.test_files.population_functions import xlsx_to_csv
+from models.user import *
+from models.team import *
+from models.team_user import *
+from models.user_course import *
+from models.course import *
+from sqlalchemy import *
+from core import db
+from datetime import date
+import csv
+
+def __uncommit_changes(
+        new_user_ids: List[int], 
+        new_user_courses: List[int], 
+        new_team_id: int, 
+        course_id: int) -> None:
+
+    """ 
+    DESCRIPTION:
+    Gets called when an error occurs when reading the roster file or something
+    went wrong with the database transaction. When called, it will delete all
+    the new users, user_courses, and team_users that were created.
+
+    PARAMETERS:
+    new_user_ids:    List[int]: A list of user_ids that were created.
+    new_user_course: List[int]: A list of *user_ids* that were created.
```

Comment:
Is this comment correct? new_user_course and new_user_ids are both lists of user_ids that were created?